### PR TITLE
Add amendment cr to lead motion slide

### PR DESCRIPTION
**Pull request by tsiegleauq**
Adds amendment cr to the lead motion slide.
Previously the amendment cr was only present in the amendment slide but was forgotten in the motion slide

### CODE REVIEW

File: client/src/app/slides/motions/motion/motion-slide.component.ts [hunk 1]
```diff
@@ -215,8 +212,14 @@ export class MotionSlideComponent extends BaseMotionSlideComponent<MotionSlideDa
...
             this.data.data.amendments.forEach(amendment => {
-                const paras = this.getAmendmentAmendedParagraphs(amendment);
-                paras.forEach(para => this.allChangingObjects.push(para));
+                if (amendment.change_recommendations?.length) {
+                    const amendmentCRData = amendment.change_recommendations;
+                    const amendmentCRs = amendmentCRData.map(cr => new MotionSlideObjChangeReco(cr));
+                    this.allChangingObjects.push(...amendmentCRs);
+                } else {
+                    const paras = this.getAmendmentAmendedParagraphs(amendment);
+                    paras.forEach(para => this.allChangingObjects.push(para));
```
Comment (FinnStutzenstein) [inline client/src/app/slides/motions/motion/motion-slide.component.ts:221]: use `concat`?

Comment (FinnStutzenstein) [reply client/src/app/slides/motions/motion/motion-slide.component.ts:221]: ...or  `this.allChangingObjects.push(...paras)` as above?

**Review by FinnStutzenstein — APPROVED** [review]
Ok for me. Not tested, code looks good.

**Review by emanuelschuetze — APPROVED** [review]

**Comment by emanuelschuetze** [comment]
Travis failed (python tests)!

**Pull request merged**